modules/nw-metallb-bgppeer-cr.adoc

```diff
@@ -45,6 +45,17 @@ If you use this field, you cannot specify a value in the `spec.peerASN` field.
 |`spec.peerAddress`
 |`string`
 |Specifies the IP address of the peer to contact for establishing the BGP session.
+If you use this field, you cannot specify a value in the `spec.interface` field.
+
+|`spec.interface`
+|`string`
+|Specifies the interface name to use when establishing a session. 
+Use this field to configure unnumbered BGP peering.
+There must be a direct, point-to-point connection between the two BGP peers, and you must enable IPv6 RAs (Router Advertisements).
```

Comment:
Here it not in the traditional sense soley a point to point interface like a serial line or ATM. But any type in which you have layer 2 and IPV6 RA functioning.